src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_sysfs.c

```diff
@@ -412,16 +412,16 @@ static ssize_t ulp_uuids_show(struct device *dev,
 
 static DEVICE_ATTR_RO(ulp_uuids);
 
-static ssize_t mig_cache_update_store(struct device *dev,
-		struct device_attribute *da, const char *buf, size_t count)
+static ssize_t mig_cache_update_show(struct device *dev,
+	struct device_attribute *attr, char *buf)
 {
 	struct xocl_dev *xdev = dev_get_drvdata(dev);
 
 	xocl_update_mig_cache(xdev);
 
-	return count;
+	return 0;
```

Comment:
This looks odd to me. It is reading but always return nothing. I understand it is triggering cache updating for Docker case.  maybe we can update the cache in ecc_status read?